eclass/ecm.eclass

```diff
@@ -277,11 +277,6 @@ BDEPEND+="
 	>=kde-frameworks/extra-cmake-modules-${KFMIN}:*
 "
 RDEPEND+=" >=kde-frameworks/kf-env-4"
-if [[ ${_KFSLOT} == 6 ]]; then
-	COMMONDEPEND+=" dev-qt/qtbase:${_KFSLOT}"
-else
-	COMMONDEPEND+=" dev-qt/qtcore:${_KFSLOT}"
-fi
```

Comment:
So, effectively just yanking the dependency for everyone? You're introducing a missing dependency for hundreds of ebuilds.

Reply:
I'll see if I can come up with something better

Reply:
Thanks, but that's something I've already had started working on locally.